recipe/meta.yaml

```diff
@@ -1,35 +1,33 @@
 {% set name = "rich" %}
-{% set version = "12.5.1" %}
-
+{% set version = "13.3.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 63a5c5ce3673d3d5fbbf23cd87e11ab84b6b451436f1b7f19ec54b6bc36ed7ca
+  sha256: 2d11b9b8dd03868f09b4fffadc84a6a8cda574e40dc90821bd845720ebb8e89c
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  skip: True  # [py<36]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation
+  skip: True  # [py<37]
 
 requirements:
   host:
     - pip
     - setuptools
     - wheel
     - python
-    - poetry >=1.0.10
-    - poetry-core
+    - poetry-core >=1.0.10
```

Comment:
1.0.0 should be enough ?
https://github.com/Textualize/rich/blob/6d30ad0f30028210124c149811cbbe2b183711f9/pyproject.toml#L48

Reply:
Better use exact pinning. Either 1.0.0 and then poetry-core from noarch will be used, or arch-specific: 1.5.1, which is available on all archs.

Reply:
Okay got it!